src/content/blog/SwitchPackageToProjRef.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Seemless switching between Package and Project references
+date: 2023-11-11
+author: "dawe"
+slug: "2023/11/11"
+---
+
+## Switching seemlessly between Package and Project references - 2023-11-11
+
+##### by dawe
+
+While doing open source development on a code base with package dependencies which are crucial to your work, you easily run into a situation where you want to adjust the package code to your needs. Reasons could be a missing feature, a bug, or just a different opinion on how things should be done.  
```

Comment:
I feel like this is a bit of complex way of saying that it sucks when your NuGet dependency has a problem. I would put it a bit more straightforward of what actually happened to me earlier this week.

Reply:
Mmh, I beg to differ on this one.
To me, it reads much nicer giving first a number of reasons/motivations and decribing our own situation in more detail in line 15.